build/linux/installer/conf/kube.conf

```diff
@@ -59,13 +59,6 @@
      keepalive true
    </match>
 
-   #custom_metrics_mdm filter plugin for perf data from windows nodes
```

Comment:
Dont we need this for legacy auth and  the scenario where windows_fluent_bit is disabled?